src/components/styles/header.css

```diff
@@ -7,10 +7,13 @@
 
 #cameron-logo {
   grid-column: 1;
+  width: 150px;
```

Comment:
Could we make this % instead of `px`? It makes it more responsive. What do you think?

Reply:
I have re-pushed as per suggestion